bench/benchmark_estimate.jl

```diff
@@ -0,0 +1,103 @@
+using ScoreDrivenModels, Distributions, BenchmarkTools, Random
+
+function simulate_GAS_1_1(D::Type{<:Distribution}, scaling::Real, ω::Vector{T}, A::Matrix{T}, 
```

Comment:
this function is the same as the one in test_estimate.jl, should probably create a test_utils.jl file and load functions from it to benchmark in order to avoid code repetition

Reply:
I would say I prefer to let it this way for now because once we create a file for functions like this it will be harder to track which function goes where.